cli/plasmo/src/features/extension-devtools/project-watcher.ts

```diff
@@ -82,7 +82,9 @@ export const handleProjectFile = async (
       return
     }
     case WatchReason.PackageJson: {
-      iLog("package.json changed, update manifest overides")
+      iLog(
+        "package.json changed, update manifest overides. You might need to restart the dev server."
```

Comment:
For this error message, we're talking to "plasmo" in one sentence and to the user in another. Either of which is fine on their own but I think it might confuse people. 

```suggestion
        "package.json changed, updating manifest overides. You might need to restart the dev server."
```